specification/src/main/asciidoc/platform/ApplicationClients.adoc

```diff
@@ -309,7 +303,7 @@ file. Note that this name is case-sensitive.
 <<a3404, Jakarta EE Application Client XML Schema Structure>> shows the structure of the Jakarta EE
 application-client XML Schema. The Jakarta EE application-client XML Schema
 is located at
-_https://jakarta.ee/xml/ns/jakartaee/application-client_9.xsd_.
+_https://jakarta.ee/xml/ns/jakartaee/application-client_10.xsd_.
```

Comment:
Note that I have this as being client_10 because client_11.xsd doesn't exist yet.